### PR TITLE
feature/uweb-6139 card article

### PR DESCRIPTION
**Pull request by dan-entous**
* [x] add card article

### CODE REVIEW

File: src/js/components/molecules/cards/card-article/card-article.models.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+/** @format */
+
+import { CardBaseProps } from 'components/molecules/cards/card-base/card-base.models';
+import { CardArticle } from '@ultimaker/ultimaker.com-model-definitions/dist/molecules/cards/CardArticle';
+
+export interface CardArticleProps extends CardBaseProps {
```
Comment (zwaardje) [inline src/js/components/molecules/cards/card-article/card-article.models.ts]: ``` export interface CardArticleProps extends CardBaseProps, CardArticle {}```

File: src/js/components/molecules/cards/card-article/card-article.scss [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+@import "~scss/settings/_variables.scss";
+
+.card-article {
+    position: relative;
+    background-color: $colorWhite;
+    border: 1px solid $colorGrayLight;
+}
+
+.card-article__heading {
+    margin-bottom: 19px;
```
Comment (zwaardje) [inline src/js/components/molecules/cards/card-article/card-article.scss]: `$buildingUnits`

[the remark this answers is not in the document]
Comment (dan-entous) [reply src/js/components/molecules/cards/card-article/card-article.scss]: ja, vertical rhythm stuff carried over from card-product

Comment (dan-entous) [reply src/js/components/molecules/cards/card-article/card-article.scss]: ja, vertical rhythm stuff carried over from card-product

File: src/js/components/molecules/cards/card-article/card-article.models.ts (hunk 1 above)
Comment (dan-entous) [reply src/js/components/molecules/cards/card-article/card-article.models.ts]: tried it out but a few errors appeared. we can look at this together a bit later.

**Pull request merged**